Lombiq.Tests.UI/Extensions/BasicOrchardFeaturesTestingUITestContextExtensions.cs

```diff
@@ -612,7 +612,11 @@ await context
 
                 context.Get(By.Id("create-folder-name")).SendKeys("Example Folder");
 
-                await context.Get(By.Id("modalFooterOk")).ClickReliablyAsync(context);
+                await context.ClickReliablyOnAsync(By.Id("modalFooterOk"));
+
+                // Wait until new folder is created.
+                context.Exists(
+                    By.XPath("//div[contains(@class, 'alert-info') and contains(.,'This folder is empty')]"));
```

Comment:
For `XPath`s I would recommend using `$` for the `string` as in other places.
```suggestion
                    By.XPath($"//div[contains(@class, 'alert-info') and contains(.,'This folder is empty')]"));
```

Reply:
Rider shows Redundant string interpolation warning while using.
![image](https://github.com/Lombiq/UI-Testing-Toolbox/assets/19938236/9d46baca-dede-420a-b2d5-0d432c975d99)